x/gov/keeper/grpc_query.go

```diff
@@ -275,6 +275,19 @@ func (q queryServer) TallyResult(ctx context.Context, req *v1.QueryTallyResultRe
 	return &v1.QueryTallyResultResponse{Tally: &tallyResult}, nil
 }
 
+// ProposerOfProposal returns the proposer of a governance proposal
+func (q queryServer) ProposerOfProposal(ctx context.Context, req *v1.QueryProposerOfProposalRequest) (*v1.QueryProposerOfProposalResponse, error) {
```

Comment:
if we have a proposal query why do we need a proposer query?

Reply:
we can merge, mainly thinking how to avoid more code to maintain in the long run.

Reply:
Yeah I understand. However that is very little code. All that generated code makes it look like it is much.
🤷‍♂️

Reply:
its more to maintain into the future if we try to rewrite things is my main gripe. Less features = less code = less to maintain. Some features are needed, but worried what could lead from this, making query requests for single values is overkill. Thats my take but wont block the merge

Reply:
I sorta agree, however I really want to delete the crappy current proposer command.
I will add an alias to proposal as proposer as suggested as you can still get the value that way.